app/views/sufia/homepage/_sortable_featured.html.erb

```diff
@@ -1,26 +1,26 @@
-<% featured = f.object.generic_work_solr_document %>
-<li class="dd-item dd3-item featured-item" data-id="<%= featured.id %>">
+<% presenter = f.object.presenter %>
+<li class="dd-item dd3-item featured-item" data-id="<%= presenter.id %>">
   <div class="dd-handle dd3-handle"><%= t :drag %></div>
   <div class="dd3-content panel panel-default">
     <%= f.hidden_field :id %>
-    <%= f.hidden_field :order, data: {property: "order"} %>
+    <%= f.hidden_field :order, data: { property: "order" } %>
     <div class="main row">
       <div class="col-sm-3">
-        <%= link_to curation_concerns_generic_work_path(featured) do %>
-          <%= render_thumbnail_tag featured, {width: 90} %>
+        <%= link_to curation_concerns_generic_work_path(presenter) do %>
```

Comment:
This is going to conflict with #2198. Do you have a preference which PR is merged first?